### DISCUSSION TITLE
Show README.md but remove it from list

### DISCUSSION THREAD

**Discussion by ayukawaa** [DirectoryLister/DirectoryLister#625] [Help & Support]
Hi,

Is it possible to DISPLAY the README.md and at the same time HIDE IT in the list of downloable files so people dont click on it to download?

**Comment by PHLAK** [comment]
This is not possible at the moment but I will look into it.

**Comment by studiomav** [comment]
Came here to ask the same thing, I still want to display md files while also hiding *.md from the file lists.

**Comment by lunofe** [comment]
I have found a solution for this:

1. Open `/app/views/index.twig`
2. Find this code:
    ```
    {% for file in files %}
        {{ include('components/file.twig') }}
    {% endfor %}
    ```
3. Replace it with:
    ```
    {% for file in files %}
        {% if file.getRelativePathname != 'README.md' %}
            {{ include('components/file.twig') }}
        {% endif %}
    {% endfor %}
    ```

Result:
![Screenshot from 2022-11-30 21-11-17](https://user-images.githubusercontent.com/42828197/204899331-2c1bb2a2-f2ce-4244-8b01-843b207a1695.png)

**Reply by jamal2362** [reply]
Thank you! It's working fine!
